src/Stratis.Bitcoin.IntegrationTests/SharedSteps.cs

```diff
@@ -52,16 +52,10 @@ public void MineBlocks(int blockCount, CoreNode node, string accountName, string
 
         public void WaitForBlockStoreToSync(params CoreNode[] nodes)
         {
-            if (nodes.Length == 1)
-            {
-                TestHelper.WaitLoop(() => TestHelper.IsNodeSynced(nodes[0]));
-                return;
-            }
+            TestHelper.WaitLoop(() => TestHelper.IsNodeSynced(nodes[0]));
```

Comment:
Stick a for loop over this so that all nodes are synced before checking them against each other.